dht.go

```diff
@@ -417,7 +412,23 @@ func (dht *IpfsDHT) Mode() ModeOpt {
 	return dht.auto
 }
 
-// fixLowPeersRoutine tries to get more peers into the routing table if we're below the threshold
+func (dht *IpfsDHT) populatePeers(_ goprocess.Process) {
+	if !dht.disableFixLowPeers {
+		dht.fixLowPeers(dht.ctx)
+	}
```

Comment:
I don't think having synchronous operation in `New` is a great idea, especially since we weren't doing it beforehand. This could be solve by just calling `fixRTIfNeeded()` after calling `dht.proc.Go(dht.fixLowPeersRoutine)`.

Separately, do you think we should be doing this at all? If the user calls some function before having properly bootstrapped their results might not be as good as if they had called `Bootstrap` first. Doing this makes it more likely people will just not bother to call Bootstrap since things seem to be working. I'm fine leaving this behavior change in though.

Reply:
It isn't synchronous, it happens inside a goroutine

Reply:
Yep, I wasn't really sold on making this happen asynchronously, but given my comments from https://github.com/libp2p/go-libp2p-kad-dht/pull/694/files#r508613135 I think this is fine for now.